shortcodes/ucf-degree-list-shortcode.php

```diff
@@ -38,6 +39,10 @@ public static function shortcode( $atts ) {
 				);
 			}
 
+			if ( $atts['post_parent'] && is_int( $atts['post_parent'] ) ) {
```

Comment:
It doesn't look like we have any shortcode argument filtering set up on this shortcode that would force `int` typing on `$atts['post_parent']`--so `$atts['post_parent']` will likely end up being a string here when set (could be worth double-checking).  If `$atts['post_parent']` is a string, you'll want to use `is_numeric( $atts['post_parent'] )` here instead of `is_int()`.